.github/ISSUE_TEMPLATE/testplan.md

```diff
@@ -1352,13 +1377,33 @@ With a default Postgres DB instance, a Teleport instance configured with DB acce
   - [ ] Verify that launching any other command that's not supported by the autocomplete opens a new
         local shell with that command running.
 - Resilience when resources become unavailable
-  - For each scenario, create at least one tab for each available kind (minus k8s for now).
-  - For each scenario, first do the external action, then click "Sync" on the relevant cluster tab.
-    Verify that no unrecoverable error was raised. Then restart the app and verify that it was
-    restarted gracefully (no unrecoverable error on restart, the user can continue using the app).
-    * [ ] Stop the root cluster.
-    * [ ] Stop a leaf cluster.
-    * [ ] Disconnect your device from the internet.
+  - DocumentCluster
+    - For each scenario, create at least one DocumentCluster tab for each available resource kind.
+    - For each scenario, first do the action described in the bullet point, then click "Sync" on the
+      relevant cluster tab. Verify that no unrecoverable error was raised (that is, the app still
+      works). Then restart the app and verify that it was restarted gracefully (no unrecoverable
+      error on restart, the user can continue using the app).
+      - [ ] Stop the root cluster.
+      - [ ] Stop a leaf cluster.
+      - [ ] Disconnect your device from the internet.
+  - DocumentGateway
+    - Create a db connection tab for a given database. Then remove access to that db for that user.
+      Go back to Connect and:
+      - Change database name in the tab. This should still work.
+      - Change database port. This should result in an error.
```

Comment:
This was written when I was still thinking that restarting a gateway is going to refetch the certs. Restarting a gateway doesn't update certs in any way, so both actions should work.

Look up the test plan for v11 to copy the updated test plan items for this section.